db/model/Gdoc/htmlToEnriched.ts

```diff
@@ -66,23 +68,35 @@ export function consolidateSpans(
 ): OwidEnrichedGdocBlock[] {
     const newBlocks: OwidEnrichedGdocBlock[] = []
     let currentBlock: EnrichedBlockText | undefined = undefined
+
     for (const block of blocks) {
-        if (block.type === "text")
-            if (currentBlock === undefined) currentBlock = block
-            else
+        if (block.type === "text") {
+            if (currentBlock === undefined) {
+                currentBlock = block
+            } else {
                 currentBlock = {
                     type: "text",
                     value: [...currentBlock.value, ...block.value],
-                    parseErrors: [],
+                    parseErrors: [
```

Comment:
Shouldn't we at least put a space here when concatenating text blocks?

```suggestion
                    value: [
                        ...currentBlock.value,
                        {
                            spanType: "span-simple-text",
                            text: " ",
                        },
                        ...block.value,
                    ],
```